tool/src/test/resources/org/datacommons/tool/genmcf/measurementresult/output/report.json

```diff
@@ -12,7 +12,7 @@
     "LEVEL_WARNING": {
       "counters": {
         "Existence_MissingTriple_domainIncludes": "1",
-        "API_EmptyDcCallResponse": "1"
+        "StatsCheck_Inconsistent_Values": "2"
```

Comment:
Is this the input CSV? https://github.com/datacommonsorg/import/blob/master/tool/src/test/resources/org/datacommons/tool/genmcf/measurementresult/input/acre.csv

Wonder why the counter is 2?

Also, can we add more data points to trigger inconsistent value types in a series?

Reply:
> Is this the input CSV?

Yes.

> Wonder why the counter is 2?

Good catch, I don't know yet, will investigate this further.

An interesting thing here from the report.json; the `lineNumber` of rows 3 and 4 are both logged as 3. Relevant section; (marked the lineNumber rows)
```
"values": [{
          "value": {
            "type": "RESOLVED_REF",
            "value": "Acre1.0",
            "column": "Acre_MeasurementResult_StatVar"
          },
          "locations": [{
            "file": "acre.csv",
            "lineNumber": "2"       <========
          }]
        }, {
          "value": {
            "type": "RESOLVED_REF",
            "value": "Acre10.0",
            "column": "Acre_MeasurementResult_StatVar"
          },
          "locations": [{
            "file": "acre.csv",
            "lineNumber": "3"       <========
          }]
        }, {
          "value": {
            "type": "RESOLVED_REF",
            "value": "Acre100.0",
            "column": "Acre_MeasurementResult_StatVar"
          },
          "locations": [{
            "file": "acre.csv",
            "lineNumber": "3"       <========
          }]
        }]
      }, {
```


> Also, can we add more data points to trigger inconsistent value types in a series?

TODO(snny)

Reply:
> Also, can we add more data points to trigger inconsistent value types in a series?

Done

Reply:
> > Wonder why the counter is 2?

> Good catch, I don't know yet, will investigate this further.

See #177 for a discussion about what is happening here and potential future directions. Marking this conversation resolved since that thread now has a place and the other request was completed.